<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackWidget</class>
 <widget class="QWidget" name="PlaybackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Playback</string>
  </property>
  <layout class="QHBoxLayout" name="playbackMainHLayout" stretch="6,4">
   <item>
    <widget class="QGroupBox" name="playbackSettingsGroup">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="playbackSettingsVLayout">
      <item>
       <layout class="QVBoxLayout" name="playbackFilenameVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="playbackFilenameLbl">
          <property name="text">
           <string>File to play</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="playbackFilenameHLayout">
          <item>
           <widget class="QLineEdit" name="playbackFilename"/>
          </item>
          <item>
           <widget class="QPushButton" name="playbackOpenFileDialog">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="playbackInterfaceVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="playbackInterfaceLbl">
          <property name="text">
           <string>Network interface</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="playbackInterfaceSelect"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="playbackAddressHLayout" stretch="2,1">
        <item>
         <layout class="QVBoxLayout" name="AddressVLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="playbackHostLbl">
            <property name="text">
             <string>Multicast address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playbackHost"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="playbackPortVLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="playbackPortLbl">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playbackPort"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="playbackExpandPCAPFilterButton">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Advanced</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="playbackPcapFilterContainer" native="true">
        <layout class="QVBoxLayout" name="playbackPcapFilterVLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="playbackFilterLbl">
           <property name="text">
            <string>PCAP file filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="playbackFilter"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="playbackFilteringHLayout" stretch="1,3">
        <item>
         <widget class="QCheckBox" name="playbackLoopCheckbox">
          <property name="text">
           <string>Loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="playbackLoopCombo" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="playbackStartStopBtn">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="playbackStatusVLayout">
     <item>
      <widget class="QGroupBox" name="playbackStreamInfoGroup">
       <property name="title">
        <string>Stream info</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="styleSheet">
           <string notr="true">QTreeView::branch:has-siblings:!adjoins-item {
    border-image: url(:/img/res/stylesheet-vline.png) 0;
}

QTreeView::branch:has-siblings:adjoins-item {
    border-image: url(:/img/res/stylesheet-branch-more.png) 0;
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:/img/res/stylesheet-branch-end.png) 0;
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/img/res/stylesheet-branch-closed.png);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:/img/res/stylesheet-branch-open.png);
}</string>
          </property>
          <column>
           <property name="text">
            <string>Streams (Selected is filtered for playback)</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="playbackStatusGroup">
       <property name="title">
        <string>Status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="playbackStatus">
          <property name="text">
           <string>Idle</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="recordFileFormatGroup"/>
 </buttongroups>
</ui>
